CONTRIBUTING.md

```diff
@@ -111,6 +124,15 @@ with an `@` sign. Example:
 
 The opening `/**` marker must be immediately followed by a line feed.
 
+#### Documentation comment placement
+Documentation comments of public procedures must be provided in the corresponding header files.
+
+Documentation comments for nontrivial private procedures must be provided where those procedures are
+implemented.
+
+Duplication of multiline comments or of sigificant parts of multiline comments is forbidden.
+Doxygen's `@implements` command should be used instead of comment duplication.
```

Comment:
A piece of example code here would be instructive.

Reply:
Actually, I just realized that I do not understand why we would do this; I just unthinkingly added this because you mentioned it earlier. It should be clear from the procedure name what prototype it implements; furthermore, the implementation of the prototype generally shouldn't need any documentation comment at all. If the implementation does have appear to have some noteworthy behavior, then that can be indicated by an inline comment that briefly explains why the behavior is consistent with the contract specified in the documentation of the prototype.

If we were to do this, I think it would appear at the top of the file, like this:
`scheduler.h`:
```
/**
 * @file
 * @interface scheduler
 * @brief Define a scheduler for the threaded C runtime.
 * @author Soroush Bateni <soroush@utdallas.edu>
 * @copyright See "LICENSE.md."
 */

...
```

`scheduler_NP.c`:
```

/**
 * @file
 * @implements scheduler
 * @brief Define a non-preemptive scheduler.
 * @author Soroush Bateni <soroush@utdallas.edu>
 * @copyright See "LICENSE.md."
 */

...
```

But why would we do this? Based on the dearth of information about `@implements` on the internet, I suspect that it is very rare.

Reply:
I started to arrive at the same conclusion. It appears that document inheritance should work out of the box if `INHERIT_DOCS` is enabled in the Doxygen configuration. It also works out of the box in VS Code. Conclusion: I don't think we need `@inherits` or `@implements`.

Reply:
Okay, I deleted `@implements` from the document.